### PR TITLE
DOC: Update conda create/export to be more explicit.

### PR DESCRIPTION
**Pull request by danielballan**

### CODE REVIEW

File: source/deployment/updating-metapackages.rst [hunk 1]
```diff
@@ -92,7 +93,7 @@ Defining Environments
 
    .. code-block:: bash
 
-      conda env export -p /tmp/analysis-YYYY-X.V -f ~/analysis-YYYY-X.V.yml
+      conda env export -p /tmp/analysis-YYYY-X.V -f ~/analysis-YYYY-X.V.yml--override-channels -c http://alexandria/conda/defaults -c http://alexandria/conda/nsls2-tag
```
Comment (mrakitin) [inline source/deployment/updating-metapackages.rst]: extra blank space before `--override-channels`?

**Review by mrakitin — COMMENTED** [review]
There is a missing blank space. Otherwise looks good.

File: source/deployment/updating-metapackages.rst (hunk 1 above)
Comment (danielballan) [reply source/deployment/updating-metapackages.rst]: Fixed the force-pushed. Thanks.

**Review by mrakitin — APPROVED** [review]
Looks good now.

**Pull request merged**